src/pe/utils.rs

```diff
@@ -51,6 +53,12 @@ fn section_read_size(section: &section_table::SectionTable, file_alignment: u32)
     // }
 
     let file_alignment = file_alignment as usize;
+    if file_alignment == 0 || file_alignment & (file_alignment - 1) != 0 {
+        return Err(error::Error::Malformed(format!(
+            "file_alignment should be a power of 2 {}",
```

Comment:
I suggest a more accurate message, such as `"file_alignment={} is not a strictly positive power of 2"`.